### PR TITLE
Blog updates for weeks 7 & 8

### PR DESCRIPTION
**Pull request by Disha-maker**
Please check the final changes Ciaran or anyone who is free now

### CODE REVIEW

File: _lifeofinternatpuppet/post_7_and_8 [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+So I will say firstly that the demo was amazing, I really enjoyed the experience and got more knowledge from someone like Norman who had 20 years experience in mentoring... Sorry Norman, I had to reveal this fact as I think anyone reading this post should be aware of this fact :)
+
+He also said that next time when I am using Litmus and running tests on a provisioned machine, I should select a difficult module/repository to clone down from GitHub, just to give muself a challenge and see if I am do the same process sucessfully!
+I will look into this later in the coming weeks so stay connected to find out more :)
+
+He also suggested that I should read up on the fundamentals of unix and bash commands via [LinkedIn Learning](https://www.linkedin.com/learning/) as it will really help me in my day-to-day work.
+
+He also gave me a breakdown on how I should start my first ticket/task on changing the author name of the supported modules to 'puppetlabs' He gave me useful advice was given on how I should approach the ticket, so thank you so much Norman for all your help to get me started off on my first ticket!ß
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_7_and_8:130]:
```suggestion
He also gave me a breakdown on how I should start my first ticket/task on changing the author name of the supported modules to 'puppetlabs' He gave me useful advice was given on how I should approach the ticket, so thank you so much Norman for all your help to get me started off on my first ticket!
```

File: _lifeofinternatpuppet/post_7_and_8 [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+Just want to wish these people best of luck with their applications for placement year/graduates roles at Puppet!
+
+This morning will be as usual with the Friday Wrap Up session. I gave an update on the work done within the IAC Team this week and thanks for the appreciation Norman as always :)
+David Schmitt sent out useful information such as the [team blog](https://puppetlabs.github.io/iac/team/status/2020/10/23/status-update.html) to help me give a summary to the rest of the teams at the Belfast Office as well as gave a mention on Sheena's demo and will attend the social hour (let's hope it's a games afternoon as decided last week)
+I'm looking forward to the social hour :)
+
+That's all the updates done from last week and this week :)
+
+Have a good weekend everyone!
```
Comment (sanfrancrisko) [inline _lifeofinternatpuppet/post_7_and_8:186]:
```suggestion
Have a good weekend everyone!

```

**Review by sanfrancrisko — COMMENTED** [review]
2 minor fixes

File: _lifeofinternatpuppet/post_7_and_8 (hunk 1 above)
Comment (Disha-maker) [reply _lifeofinternatpuppet/post_7_and_8:130]: So should I make the change and then just push the changes?

Comment (sanfrancrisko) [reply _lifeofinternatpuppet/post_7_and_8:130]: You can just use the 'Commit suggestion' functionality here

Comment (Disha-maker) [reply _lifeofinternatpuppet/post_7_and_8:130]: There is no & symbol after the word ticket!

Comment (sanfrancrisko) [reply _lifeofinternatpuppet/post_7_and_8:130]: This suggestion above is to remove the `ß` symbol

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**